mappings/net/minecraft/server/util/PlayerFormatFixer.mapping

```diff
@@ -0,0 +1,27 @@
+CLASS none/mq net/minecraft/server/util/PlayerFormatFixer
+	CLASS none/mq$a ConversonError
+		METHOD <init> (Ljava/lang/String;)V
+			ARG 0 message
+		METHOD <init> (Ljava/lang/String;Ljava/lang/Throwable;)V
+			ARG 0 message
+			ARG 1 cause
+	FIELD a BANNED_IPS Ljava/io/File;
+	FIELD b BANNED_PLAYERS Ljava/io/File;
+	FIELD c OPS Ljava/io/File;
+	FIELD d WHITE_LIST Ljava/io/File;
+	FIELD e LOGGER Lorg/apache/logging/log4j/Logger;
+	METHOD a readFile (Ljava/io/File;Ljava/util/Map;)Ljava/util/List;
+		ARG 0 file
+		ARG 1 cache
+	METHOD a fixBannedPlayers (Lnet/minecraft/server/MinecraftServer;)Z
+		ARG 0 server
+	METHOD a (Lnet/minecraft/server/MinecraftServer;Ljava/util/Collection;Lcom/mojang/authlib/ProfileLookupCallback;)V
+		ARG 0 server
+	METHOD b mkdir (Ljava/io/File;)V
+		ARG 0 directory
+	METHOD b fixIPBans (Lnet/minecraft/server/MinecraftServer;)Z
```

Comment:
`fixIpBans`

If you go with `Upgrader`, probably call this `upgradeIpBans`, similar to the others.